lib/containers/common.pm

```diff
@@ -96,6 +96,17 @@ sub install_docker_when_needed {
     assert_script_run('docker info');
 }
 
+sub install_buildah_when_needed {
+    my $host_os = shift;
+    my @pkgs    = qw(buildah);
+    if (script_run("which buildah") != 0) {
+        # We may run openSUSE with DISTRI=sle and opensuse doesn't have SUSEConnect
+        add_suseconnect_product('sle-module-containers') if ($host_os =~ 'sles' && is_sle('>=15'));
+        zypper_call "in @pkgs";
+        assert_script_run('buildah info');
```

Comment:
maybe we can put this into a frame
`record_info('buildah', script_output('buildah info'));`

Reply:
done